### ISSUE TITLE
 [Tool Execution Fail] Btlejack

### ISSUE THREAD

**Issue by s7x** [BlackArch/blackarch#2170]
Hello there!

When I try to run btlejack, I have the following error:

```
pkg_resources.DistributionNotFound: The 'enum34==1.1.6' distribution was not found and is required by log-symbols
```

Tried installing the python-log_symbols & python-log-symbols packages but it doesn't change anything.
I also have installed the python-enum-compat and python2-enum34 packages installed.

Any idea?

Cheers & thanks!

EDIT: typos

**Comment by noptrix** [comment]
@s7x Thanks for letting us know. This is a bit strange. I tried to trace back up to 'pkg_resources' but couldn't find the issue yet related to enum34. I will try to fix it soon and let you know.

**Comment by s7x** [comment]
@noptrix Yeah I don't really get this error too. Tried debugging it on my own but failed...
Thanks ;)

**Comment by deraffe** [comment]
I ran into the same error, rebuilt the packages (including all of `python-halo`'s dependencies) in a clean environment using `extra-x86_64-build`, and it magically fixed the error. I guess this is just an artifact of a Python installation full of cruft. Though I'm not sure if it's mine or the one on the build server.

**Comment by ikstream** [comment]
This problem should be fixed with c723f6b1f27841c0f73a7efc873f3c46994080e2 now
